### ISSUE TITLE
Remove projects/models file

### ISSUE THREAD

**Issue by uhlissuh** [mozilla/experimenter#2114]
I don't think we need any files related to projects, do we?

**Comment by krtb** [comment]
Hi, can I work on this issue?

**Comment by uhlissuh** [comment]
@krtb , sure, i think first step is getting the development environment set up, are you there yet?

**Comment by krtb** [comment]
No, but would the be accomplished by following the installation steps on the README.md file?

**Comment by uhlissuh** [comment]
@krtb, yep give that a shot!

**Comment by krtb** [comment]
I was able to get the development environment setup and have it up and running on my localhost

**Comment by uhlissuh** [comment]
@krtb great, well done. This ticket was just about if we could delete the projects/models folder, which contains migrations, and not ruin things in the process. Try deleting it and then running `make check` in the terminal to see if all the tests still pass!

**Comment by krtb** [comment]
Hi @uhlissuh, I tried removing the projects folder in a new branch and running `make check`, which results in an error: `ModuleNotFoundError: No module named 'experimenter.projects'`.

The issue appears to relate to the migration file in  `app/experimenter/experiments/migrations/0002_auto_20171115_1918.py`, where it references a migration from projects folder that gets deleted `("projects", "0003_auto_20170630_1924")`, and `project` in the operations section of that file. Also it's referenced in `app/experimenter/settings.py` under installed apps.

Is there a proper way to remove the migrations that reference the deleted `"projects"` folder and rerun so that they sync up with the database? I've attempted different ways of doing this on test branches but end up with `AttributeError: 'NoneType' object has no attribute 'is_relation'`
